### PR TITLE
Intel ME

### PR DESCRIPTION
**Pull request by Ampersandstorm**
This article was created based on a response in this issue: https://github.com/system76/docs/issues/1023

### CODE REVIEW

File: content/intel-me.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+The [Intel Management Engine](https://en.wikipedia.org/wiki/Intel_Management_Engine) is a proprietary, mostly undocumented, firmware system that provides many extraneous features that are generally not usable or useful to our users, with multiple known vulnerabilities that compromise the entire system.
+
+## What is System76 Open Firmware?
+
+System76 Open Firmware is an open source distribution of firmware utilizing coreboot, EDK2, and System76 firmware applications. System76 Open Firmware can disable the IME, [among other features](/articles/transition-firmware).
+
+## Is the IME Disabled On My System76 PC?
+
+For Open Firmware systems, the IME is typically disabled by default unless doing so would break functionality (such as suspend/resume). System76 maintains a list of machines that ship with Open Firmware [in this article](/articles/open-firmware-systems/).
```
Comment (jacobgkau) [inline content/intel-me.md:32]: To better answer the question, you could also mention that the firmware menu -> Firmware Configuration Information page will show whether the IME is enabled or disabled.

Example picture (you don't need to use this, but just to show how obvious it is on that page):

me-enabled

File: content/intel-me.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: software
+tableOfContents: true
+---
+
+System76 laptops and desktops with Open Firmware will typically have the Intel Management Engine (IME) disabled, unless doing so breaks functionality. Machines running Open Firmware with coreboot can disable the IME manually.
```
Comment (thomas-zimmerman) [inline content/intel-me.md:20]: I would be a bit softer on when we disable IME:

System76 laptop will have the Intel Management Engine (IME) disabled when it does not break functionality. System76 open firmware system do have some control to enable and disable the IME as [documented here](https://github.com/system76/firmware-open/blob/master/docs/intel-me.md)

File: content/intel-me.md (hunk 1 above)
Comment (Ampersandstorm) [reply content/intel-me.md:32]: Updated these instructions. Thanks for the heads up.

File: content/intel-me.md (hunk 2 above)
Comment (Ampersandstorm) [reply content/intel-me.md:20]: Implemented your suggested wording.

**Review by thomas-zimmerman — APPROVED** [review]
Let's get this out. Looks good.

**Review by jacobgkau — APPROVED** [review]

**Pull request merged**